src/core/eer_file.cpp

```diff
@@ -225,13 +225,14 @@ void EerFile::DecodeToFloatArray(int start_eer_frame, int finish_eer_frame, floa
     /*
 	 * Decode into a list of events
 	 */
-    unsigned int*  positions = new unsigned int[max_electrons];
-    unsigned char* symbols   = new unsigned char[max_electrons];
-    for ( int iframe = start_eer_frame; iframe <= finish_eer_frame; iframe++ ) {
+   // wxPrintf("max electrons = %llu\n", max_electrons);
+    unsigned int *positions = new unsigned int[max_electrons]();
+    unsigned char *symbols   = new unsigned char[max_electrons]();
+    for ( int iframe = start_eer_frame; iframe < finish_eer_frame; iframe++ ) {
         long long     pos     = frame_starts[iframe];
         unsigned int  npixels = 0, nelectrons = 0;
         unsigned int  bit_pos = 0; // 4 K * 4 K * 11 bit << 2 ** 32
-        unsigned char rle, subpixel;
+        unsigned char rle, subpixel = 0;
```

Comment:
A multi line declaration with partial initialization isn't very clean.

Alternatives
```cpp
unsigned char rle;
unsigned char subpixel = 0;
```

or better yet, eliminate the variable subpixel. It is only being assigned to symbols[nelectrons] w/o intermediate use.